src/linux/backend-v4l2.cpp

```diff
@@ -52,6 +52,7 @@
 #pragma GCC diagnostic ignored "-Woverflow"
 
 const size_t MAX_DEV_PARENT_DIR = 10;
+const int DEFAULT_KPI_FRAME_DROPS_PCT = 5;
```

Comment:
Please add a comment " We allow 5% ..."

Reply:
changed to DEFAULT_KPI_FRAME_DROPS_PERCENTAGE and to double, with value 0.05